app/scripts/views/form.js

```diff
@@ -119,22 +118,33 @@ define(function (require, exports, module) {
      */
     getFormValues () {
       var values = {};
-      var inputEls = this.$('input,textarea,select');
 
-      for (var i = 0, length = inputEls.length; i < length; ++i) {
-        var el = $(inputEls[i]);
-        // elements that have data-novalue (like password show fields)
-        // are not added to the values.
-        if (typeof el.attr('data-novalue') === 'undefined') {
-          var name = el.attr('name') || el.attr('id');
```

Comment:
This was a bug earlier. If an element had neither a name nor an id, then the value was saved with a key of `undefined.